src/components/user/tour/tour.js

```diff
@@ -1,6 +1,24 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
+import ToutApi from "../../../api/tour-client";
```

Comment:
Typo: `ToutApi` should be `TourApi`

Try to keep the code clean. If you need to use something, then import it. Here you imported TourApi but didn't use it. 

Make sure to not have warnings in the terminal, when react app running:
```
Compiled with warnings.

src/components/user/tour/tour.js
  Line 3:8:  'ToutApi' is defined but never used  no-unused-vars
```